GDPR-Phase2-Java/src/main/java/com/gdpr/phase2/Utils/GVL.java

```diff
@@ -0,0 +1,539 @@
+package com.gdpr.phase2.Utils;
+
+import com.gdpr.phase2.model.ConsentLanguages;
+import com.gdpr.phase2.model.gvl.*;
+
+
+
+import java.util.*;
+
+
+enum PurposeOrFeature {
+    Purpose(0),
+    Feature(1);
+
+    private int type = -1;
+
+    PurposeOrFeature(int type) {
+        this.type = type;
+    }
+
+    public int getType() {
+        return type;
+    }
+
+    public void setType(int type) {
+        this.type = type;
+    }
+}
+
+enum PurposeSubType {
+    consent(0),
+    legInt(1),
+    flexible(2);
+
+    private int type = -1;
+
+    PurposeSubType(int type) {
+        this.type = type;
+    }
+
+    public int getType() {
+        return type;
+    }
+
+    public void setType(int type) {
+        this.type = type;
+    }
+}
+
+/**
+ * class with utilities for managing the global vendor list.  Will use JSON to
+ * fetch the vendor list from specified url and will serialize it into this
+ * object and provide accessors.  Provides ways to group vendors on the list by
+ * purpose and feature.
+ */
+
+/**
+ * @Usage
+     * My Library is right now only implementing following cunstructor
+         @Constructor GVL(GVL vendorList)
+     * Promises are Ignored in the GVL loading for now
+
+     * IntMap class has been removed in this Implementation for the proper and simple JSON parsing of GVL json
```

Comment:
It was creating problems when i am trying to parse the JSON. Thats why i have to remove it and implement the direct maps. Talking about generic part, Now instead of creating a generic IntMap class object we can rather create direct maps for whatever key and value pair we want. 
P.S. the internal structure of IntMap was **[id: string]: T** Now it is Map<String,T> where T is specific to where we are creating the type.